### DISCUSSION TITLE
Start nginx Service cant work raghu garu

### DISCUSSION THREAD

**Discussion by chowdary709** [learndevopsonline/learndevopsonline#1869] [Q&A]
![Screenshot (4)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/cadbe533-cf58-40ea-85a5-2aa24cb2f357)
![Screenshot (6)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/e0d0235b-6d28-4407-8ef9-0b37c635bc23)

**Comment by chaithanya1812** [comment]
I am unable to login into server,
@chowdary709  login into frontend server, try to start the frontend service, & post the error.

## @chowdary709 , what happened to your code??
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/4384def5-0d90-48fc-9b4e-5472db8d2a65)

**Comment by chowdary709** [comment]
working
